Lib/ctypes/util.py

```diff
@@ -94,10 +94,13 @@ def find_library(name):
     import re, tempfile
 
     def _is_elf(filename):
-        "Return True if the given file is an ELF file"
+        "Return True if the given file is readable and is an ELF file"
```

Comment:
I think readable was already implied: if you can’t read the file you can’t determine its type!

Reply:
updated. Thanks!